examples/java/Java13Example.java

```diff
@@ -1649,18 +1472,17 @@ public void treeExample(Context ctx) throws TestFailedException
     // / forest ::= nil | cons(tree, forest)
     // / tree ::= nil | cons(forest, forest)
     // / </remarks>
-    public void forestExample(Context ctx) throws TestFailedException
+    @SuppressWarnings({"unchecked", "unused", "UnusedAssignment"})
+    public <Tree, Forest> void forestExample(Context ctx) throws TestFailedException
```

Comment:
I especially encourage you to look at this to get a better understanding of why these changes are such a quality of life improvement.